adafruit_ltr390.py

```diff
@@ -65,7 +72,11 @@ def __init__(self, register_address, struct_format, bitwidth, length):
         self._width = bitwidth
         self._num_bytes = length
 
-    def __get__(self, obj, objtype=None):
+    def __get__(
+        self,
+        obj: Optional["UnalignedStruct"],
```

Comment:
The way `__get__()` and `__set__()` work with the `obj` argument (or more technically the first non-self argument) is that it's the object who has an instance of this class as an attribute, like it is used here:  https://github.com/tcfranks/Adafruit_CircuitPython_LTR390/blob/d13a609be095a2fbf0aa548ffac163bc712e1dff/adafruit_ltr390.py#L277

So your uses of `"UnalignedStruct"` should, for the way it's used in this library, be `"LTR390"`.  That goes for both here, and the one in `__set__()`